pyproject.toml

```
@@ -40,3 +40,11 @@ fail_under=40.0
 [tool.coverage.html]
 
 show_contexts=true
+
+[tool.isort]
+profile = "black"
+py_version=310
+sections = ['FUTURE', 'STDLIB', 'TYPING', 'THIRDPARTY', 'FIRSTPARTY', 'LOCALFOLDER']
+known_typing = ["typing", "types"]
```

Not saying we have to, but I suspect code I wrote will have a much more minimal diff with
```suggestion
sections = ['FUTURE', 'TYPING', 'STDLIB', 'THIRDPARTY', 'FIRSTPARTY', 'LOCALFOLDER']
known_typing = ["typing", "types", "collections.abc"]
```

---

Adopted and run (looks like a reasonable ordering to me).